$NetBSD: patch-am,v 1.2 2015/12/06 11:35:55 wiz Exp $

on darwin configure script picks up private per-user tmp directory
in /var/folders/xx/...+++... via the TMPDIR environment and cheerfully
installs it in /usr/pkg/etc/dbus-1/session.conf meaning that only the
user who installed the package will be able to write to that directory...
except that dbus rejects directories with "+"'s in the filename so it
won't work anyway.   use configure --with-session-socket-dir=/tmp
to work around this.   found a description of this issue here:
http://wiki.gnucash.org/wiki/MacOSX/Quartz in the
"Running from the commandline" section.

dbus also cannot cope with $DISPLAY containing "/" chars (like in
darwin ... /tmp/launch-kcvznx/:0 ) so get rid of the ":" as described
here:
 https://trac.macports.org/attachment/ticket/16833/patch-dbus-launch-x11.c.diff

--- tools/dbus-launch-x11.c.orig	2008-11-14 22:45:04.000000000 -0500
+++ tools/dbus-launch-x11.c	2008-11-14 22:45:44.000000000 -0500
@@ -143,7 +143,7 @@
    */
   for (p = display; *p; ++p)
     {
-      if (*p == ':')
+      if (*p == ':' || *p == '/')
         *p = '_';
     }
   
